### PR TITLE
change functions to arrow functions

### PR DESCRIPTION
**Pull request by hasnatali1947**
- Refactor your functions to arrow functions.
- used const and let in the correct way.
- set current date and time.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @hasnatali1947 ,
 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of ESC6 syntax :clap: 
- Good use of GitHub flow :+1: 

## Required Changes ♻️
- Kindly make sure you use [luxon](https://moment.github.io/luxon/#/install?id=es6) library to meet the project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=For%20this%20requirement%20you%20should%20use%20a%20library%20Luxon)
Screenshot from 2023-03-27 12-00-20

-  Dividing your code into modules, kindly make sure the modules folder is named `modules` instead of the script to meet the project requirement and make your project better :recycle: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @hasnatali1947 ,

Good job implementing most of the requested changes!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Kindly make sure your script directory is named modules as requested by the previous reviewer. 

pr

## [Optional] suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js [hunk 1]
```diff
@@ -27,23 +26,23 @@ const currentTime = () => {
...
+const displayBooks = () => {
   container.innerHTML = '';
   bookList.books.forEach((books, index) => {
     container.innerHTML += `<ul>
     <li>"${books.title}" by ${books.author}</li>
     <li><button class = "remove" onlick= "removebook()" data-index="${index}">Remove</button></li>
     </ul>`;
   });
-}
+};
```
Comment (didierganthier) [inline index.js:37]: [OPTIONAL] Nice work overall 💯 but in order to improve it you could consider showing a message when there are no books 😉 this is what it shows 👇🏽 

Screenshot 2023-03-27 at 07 38 37

**Review by didierganthier — APPROVED** [review]
Hi @hasnatali1947 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- [ ] Well done on using the right README template 👍🏽 but the `live` link is broken for now and it can mislead, kindly replace it with the right one or mention in the README that the project hasn't been deployed. 

- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**